### ISSUE TITLE
1.18.2 | Feature Request.

### ISSUE THREAD

**Issue by Suinalk** [sgtcaze/NametagEdit#736]
Feature Example. (for mor advanced stuff config)
Example: I can change 

Haven't really started using the plugin, so the priority is probably wrong, I went higher number higher priority for this example.

groups.yml:

Groups:
  Admin:
    Permission: nte.admin
    Playername: &4{playername}
    Prefix: '&c'
    Suffix: '&f'
    SortPriority: 69
  Moderator:
    Permission: nte.moderator
    Playername: &c{playername}
    Prefix: '&2'
    Suffix: '&f'
    SortPriority: 10
  Prisoner:
    Permission: nte.prisoner
    Playername: &6{PrisonerID}
    Prefix: '&2'
    Suffix: '&f'
    SortPriority: 1
  Nickname:
    Permission: nte.nickname
    Playername: {essentialsnickname}
    Prefix: '&2'
    Suffix: '&f'
    SortPriority: 5

**Comment by Suinalk** [comment]
Basically a way to choose whether you have playername or not, or what you want in the playername :)

**Comment by MiDeng7** [comment]
Good idea. I tried to use %cmi_user_nickname% today to display the gradient player name. It can be displayed normally when added to "prefix", but the original player name will still be displayed. I didn't find the relevant function.

**Comment by Suinalk** [comment]
Could you please add it? because that will give this plugin really what it needs, being able to customize the tag nore :p

**Comment by tristiisch** [comment]
The problem with this is that it will only be displayed in the tab and not above the player. In order for it to be above the player, you have to use the same nickname (or change the player's nickname directly with a custom nickname)

**Comment by Suinalk** [comment]
I just need to hide the tag above the player... :/

**Comment by Suinalk** [comment]
I'll find a way.

**Comment by tristiisch** [comment]
Okay, but this is not the principle of this plugin, and right now, we already have to find volunteers to update the plugin so I don't think this kind of functionality will happen here. You can find another compatible plugin I think that will do what you want

**Comment by Suinalk** [comment]
Found one :33

**Comment by MiDeng7** [comment]
What did you found? Can you share it，Thanks！

**Comment by Suinalk** [comment]
I found a plugin capable of editing the nametag. (Hiding it or changing it)

**Comment by MiDeng7** [comment]
What plugin? awa

**Comment by Suinalk** [comment]
Um shit I forgot hahaha do you need it too xDD

**Comment by MiDeng7** [comment]
Yes, I need it very much QAQ

**Comment by Suinalk** [comment]
You could join my discord server and tomorrow after I get home I can check what plugin it is and I'll give the name of it ;3 because I can't do anything right now.

**Comment by Suinalk** [comment]
I will sleep now, just contact me tomorrow, join here Discord.gg/GR8V8dV (I don't really use GitHub at all that's why I'm giving you my discord) :3

**Comment by MiDeng7** [comment]
Ok, thank you
